sqlserver/check.py

```diff
@@ -324,10 +382,12 @@ def open_db_connections(self, instance):
         ]
 
         try:
-            rawconn = adodbapi.connect(self._conn_string(instance=instance),
-                                       timeout=timeout)
-            self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK,
-                               tags=service_check_tags)
+            if self._get_connector(instance) == 'adodbapi':
```

Comment:
It really shouldn't matter.  The default is adodbapi, and it's set (never left unset) in __init__.  So, I could change it, but it shouldn't matter.